compose/magento-2/bin/start

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+set -eu
```

Comment:
Any error during execution of `docker-compose up -d` (for example, error: port is already allocated) will skip starting of mutagen daemon and its sessions. 

`bin/stop` won't be able to stop containers which were started after that.

Reply:
As long as bin/stop works regardless of mutagen running already or not, would you consider this a problem? My reasoning here would be if docker compose failed, mutagen shouldn't be started (if it ends up run twice due to re-running bin/start mutagen will have duplicate sessions)

Reply:
Absolutely not a problem with working `bin/stop`. Just wanted to show one of use cases why mutagen may not start its daemon.

Reply:
Makes sense. PR updated via commit 3e3eb9e